requirements/ci.txt

```diff
@@ -46,7 +46,7 @@ code-annotations==1.3.0
     # via
     #   -r requirements/base.txt
     #   edx-lint
-coverage==6.4.4
+coverage==6.5.0
```

Comment:
Update in coverage class settings. May cause coverage drop in branch based coverage checks.